src/pages/debug-app/debug-app.tsx

```diff
@@ -828,7 +829,7 @@ export const DebugApp = () => {
     return (
       <ControlPanel
         debugMode
-        name={name}
+        mainTileset={mainTileset}
```

Comment:
I guess `mainTileset` -> `tileset` will be better naming. We don't have any not main tilesets.